### ISSUE TITLE
Scripting feature request: Add ability to change/switch color schema for the editor

### ISSUE THREAD

**Issue by macsunmood** [pbek/QOwnNotes#2647]
Add a way to switch color schema for the editor, the one defined in the Settings.
Also, ability to fetch the schema variables definitions would be very useful.

**Comment by pbek** [comment]
...another thing exactly one person needs. 😆

**Comment by macsunmood** [comment]
> ...another thing exactly one person needs. 😆

I have to disagree, since scripting is logically to be related with UI/UX tweaks including theming, etc.
And how do you know only one person would like to have such functionality?
